nestkernel/archiving_node.cpp

```diff
@@ -401,7 +401,7 @@ nest::Archiving_Node::decay_synaptic_elements_vacant( double_t p )
   {
     z = ( it->second ).get_z();
     z_vacant = ( it->second ).get_z_vacant();
-    it->second.set_z( z - ( z_vacant * p ) );
+    it->second.set_z( z - ( z_vacant * it->second.get_tau_vacant() ) );
   }
```

Comment:
@sdiazpier All quantities entering the operation in this loop are members of the `SynapticElement` object `it->second`. So I'd suggest to just call here `it->second.update_z()`, which should then look something like 
```C++
{ z_ -= get_z_vacant() * tau_vacant_; }
```
Or have I overlooked something?